config/server.json

```diff
@@ -8,5 +8,6 @@
   },
   "mongo": {
     "connectionString": "mongodb://localhost/streams"
-  }
+  },
+  "schemaVersionGte": 1
```

Comment:
I assume this means greater-than-or-equal-to, yes?  Is this for pre-UTC or post-UTC deploy?  We may need to have a schema version min and max.  Probably should discuss.